### DISCUSSION TITLE
Problem setting up MG/SAIC CUSTOM INTEGRATION with MG4

### DISCUSSION THREAD

**Discussion by Grumpie-Rich** [ad-ha/mg-saic-ha#78] [General]
I have loaded MG/SAIC CUSTOM INTEGRATION. and setup the configuration correctly as far as I know but not all of the actions work.
I try to turn on the heated seats but nothing happens and the level is not selectable, same for charge limit.

![image](https://github.com/user-attachments/assets/c424e3c3-8ea7-44be-a0d1-bc5c61502b76)

If I try to carry out the action in the development screen it comes up with an error.

![image](https://github.com/user-attachments/assets/03690452-bdc7-4f2f-86a7-33a85d19558e)

If I carry out these actions in the MG-i smart app it asks for a PIN number first.
Should this PIN number be entered in the configuration somewhere

**Comment by Grumpie-Rich** [comment]
This is the screen on the MG smart app that I have to enter a PIN

![image](https://github.com/user-attachments/assets/e763cf58-06e9-452c-af09-0dd3a80b5903)

**Comment by townsmcp** [comment]
To answer this question, in iSmart, click the last bottom button to the right,  choose Security Settings, turn off Pin Code Verification
